csharp/ql/test/library-tests/dataflow/arrays/ArrayFlow.cs

```diff
@@ -0,0 +1,229 @@
+// semmle-extractor-options: /r:System.Linq.dll
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public class A
+{
+    public void M1()
```

Comment:
Do you think it would make sense to give each of these methods a meaningful name? For example, `ArrayElementFlow`

Reply:
I can try ;-)